patches/@oclif+dev-cli+1.22.0.patch

```diff
@@ -0,0 +1,146 @@
+diff --git a/node_modules/@oclif/dev-cli/lib/commands/pack/win.js b/node_modules/@oclif/dev-cli/lib/commands/pack/win.js
```

Comment:
This patch files patches the `@oclif/dev-cli` package so that the `oclif-dev pack:win` command can run on Windows. (`oclif` was originally designed so `pack:win` "cross builds" on Linux with a bash shell, and would fail to run on Windows.) It was necessary or desirable for `pack:win` to run on Windows in order for "npm install" to build native Node.js modules for Windows.